spec/queries/consumer_apps/rpush_app_query_spec.rb

```diff
@@ -15,5 +15,16 @@
       expect(rpush_app).to be_instance_of(Rpush::Apns2::App)
       expect(rpush_app.name).to eq(consumer_app.app_bundle)
     end
+
+    it "returns nil if ConsumerApp is not operational" do
+      bad_consumer_app = create(:consumer_app, auth_key: nil)
```

Comment:
Bad consumer app, no soup for you!

![image](https://user-images.githubusercontent.com/47985/131438410-df06c5c8-1a18-491a-ac89-0df3300f6d85.png)